### PR TITLE
y

### PR DESCRIPTION
**Pull request by sergiocarracedo**
- **feat: inputfield internal component**
- **chore: wip**
- **chore: select to use inputfield**
- **feat: textarea, input, number**
- **fix: decouple upsell dialog from button (#2209)**
- **chore: release main (#2210)**
- **feat: add skeleton while product banner data is loading**
- **chore: release main (#2215)**
- **chore: disable chromatic until reviewed (#2219)**
- **fix: data collection defaultOpenGroups as true (#2220)**
- **chore: release main (#2222)**
- **fix: set correct selectable count when there is grouping (#2223)**
- **chore: release main (#2224)**
- **feat(Icons): new icons missing from Gamma (#2221)**
- **chore: release main (#2225)**
- **fix: update BasicTextEditor and added drag extension (#2212)**
- **chore: release main (#2226)**
- **fix: dialog appearing behind modal in popover context**
- **chore: release main (#2230)**
- **fix(avatar): add no translate attribute for avatars (#2228)**
- **chore: release main (#2231)**
- **chore: experimental lint (#2010)**
- **chore: focus ring**
- **feat(datacollection(): idProvider for append mode (infinite scroll) (#2182)**
- **feat: Extract datacollection filters to OneFilterPicker component (#2226) (#2232)**
- **chore: release main (#2235)**
- **feat: new module icons (#2236)**
- **chore: release main (#2237)**
- **chore: fix circular imports**
- **chore: vitest ui (#2234)**
- **feat: make label mandatory**
- **feat: make label mandatory**
- **chore: merge fixes**
- **fix: numericinput hover padding**
- **fix: numericinput hover padding**
- **refactor: extract useDataCollection and useData from datacollection**
- **docs: wip**
- **refactor: extract specific datacollection features from useDatasource to useDataCollectionSource**
- **refactor: search Options**
- **chore: wip**
- **chore: wip**
- **refactor: useDataCollectionData**
- **chore: fix types**
- **chore: fix types**
- **chore: fix types**
- **chore: fix types**
- **chore: fix types**
- **chore: fix types**
- **chore: fix types**
- **chore: fix tests**

## Screenshots (if applicable)

[Link to Figma Design](Figma URL here)

### CODE REVIEW

File: .cursor/rules/02-typescript-testing.mdc [hunk 1]
```diff
@@ -22,7 +21,10 @@ alwaysApply: false
 - Test component variants and edge cases.
 - Avoid excessive mocking; test behavior, not internals.
 - Always run tests to confirm changes.
+  <<<<<<< HEAD
+  =======
 - <IMPORTANT> When you are using `@testing-library/react` don't use `render`, use `zeroRender` from `@/testing/test-utils.tsx` instead as this provides mocks for the context providers.</IMPORTANT>
+  > > > > > > > feat/selelect-use-datasource
```
[the remark this answers is not in the document]
Comment (sergiocarracedo) [reply .cursor/rules/02-typescript-testing.mdc:27]: oh, the linter doesnt check markdown files.

Good catch!!
